3rdParty/immer/v0.6.2/tools/include/nonius.h++

```diff
@@ -659,7 +659,7 @@ namespace nonius {
             }
         };
         template <typename Sig>
-        using ResultOf = typename std::result_of<Sig>::type;
+        using ResultOf = typename std::invoke_result<Sig>::type;
```

Comment:
Is that actually correct?

Reply:
Good catch - probably not! To make this correct we would have to change the template signature.

Reply:
Fixed.